dev/CommandBarFlyout/CommandBarFlyout.xaml

```diff
@@ -295,16 +295,18 @@
                                         <ColumnDefinition Width="*" />
                                         <ColumnDefinition Width="Auto" />
                                     </Grid.ColumnDefinitions>
-                                    <ItemsControl x:Name="PrimaryItemsControl"
-                                        Height="40"
-                                        Grid.Column="0"
-                                        IsTabStop="False">
-                                        <ItemsControl.ItemsPanel>
-                                            <ItemsPanelTemplate>
-                                                <StackPanel Orientation="Horizontal" />
-                                            </ItemsPanelTemplate>
-                                        </ItemsControl.ItemsPanel>
-                                    </ItemsControl>
+                                    <Grid MaxWidth="400"
```

Comment:
Where is '400' coming from? Should this be a TemplateBinding to the parent control's MaxWidth?

Reply:
The more button has a width of 40, and the control at large has a width of 440, so 440 - 40 = 400.

If you'd like, I could update the TemplateSettings object to contain this value.  Since it's not going to change with the state of the control, and since you'd need to retemplate CommandBarFlyoutCommandBar to change its appearance anyway if you wanted to, I figured just leaving it as a hardcoded value would be easier.  (I could also add a comment about where the number comes from.)

Reply:
Adding a comment should be sufficient.

Reply:
Actually, I did some more digging and found that this was actually hiding another issue, which is that the value of TemplateSettings.CurrentWidth is not taking MaxWidth into account, whereas it should, since it's being applied as the width of a root element.  Making that change, plus moving some setters up to OuterContentRoot (so ContentRoot is not constrained when we measure), plus changing the structure of PrimaryItemsRoot so the column with the ItemsControl is a star column instead of an Auto column (so it *does* get constrained when measuring the whole thing) made this hard-coded value no longer necessary.